core/templates/dev/head/collection_editor/CollectionEditor.js

```diff
@@ -20,7 +20,7 @@
 oppia.constant(
   'COLLECTION_DATA_URL_TEMPLATE', '/collection_handler/data/<collection_id>');
 oppia.constant(
-  'WRITABLE_COLLECTION_DATA_URL_TEMPLATE',
+  'COLLECTION_EDITABLE_DATA_URL_TEMPLATE',
```

Comment:
Everywhere else you say "editable collection" rather than "collection editable". Prefer to be consistent.